portalnet/src/overlay_service.rs

```diff
@@ -398,7 +398,11 @@ where
         command_tx
     }
 
-    fn add_bootnodes(&mut self, bootnode_enrs: Vec<Enr>) {
+    /// Insert a vector of enr's into the routing table
+    /// set_connected: should only be true for tests, false for production code
+    /// Tests what use this function are testing if adding to queue's work not if our connection code
```

Comment:
`Tests that use this function are testing if adding to queues works, not if our connection code works.`